Formula/camlp5.rb

```diff
@@ -21,6 +22,7 @@ class Camlp5 < Formula
   end
 
   depends_on "ocaml-findlib" => :build
+  depends_on "camlp-streams"
```

Comment:
Alternatively, might just add as `resource` and install as part of `camlp5` rather than adding a new formula. In this case, build will probably require `ENV["OCAMLPATH"] = lib/"ocaml"` since it isn't linked at buildtime.